apps/nova/src/components/Common/Animate.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
```

Comment:
There is a component called [`Animate`](https://github.com/litespace-org/litespace/blob/master/packages/luna/src/components/Animate/Animate.tsx) which used to animate the opacity and the height of the children. You can find how it is being used in the `TopicSelectionDialog`.


You can create component and call it `AnimateOpacity` in `luna/src/components/Animate/AnimateOpacity.tsx` and re-use it here.